pennylane/ops/qubit.py

```diff
@@ -772,7 +772,7 @@ class QubitStateVector(Operation):
     num_params = 1
     num_wires = All
     par_domain = "A"
-    grad_method = None
+    grad_method = None  # FIXME 'F'?
```

Comment:
Should this operation be differentiable? F should work in principle, but there is no guarantee the norm of the state vector remains 1. Same question with `QubitUnitary`.

On the CV side, both `FockStateVector` and `FockDensityMatrix` are F-differentiable.
`QubitDensityMatrix` does not seem to exist.

Reply:
> F should work in principle, but there is no guarantee the norm of the state vector remains

I believe this was the main reason they were made `None`, instead of `'F'`.